shared/p2p/options_test.go

```diff
@@ -1,11 +1,51 @@
 package p2p
 
 import (
+	"io/ioutil"
+	"os"
 	"testing"
+
+	crypto "github.com/libp2p/go-libp2p-crypto"
+	"github.com/libp2p/go-libp2p/config"
+	"github.com/prysmaticlabs/prysm/shared/testutil"
 )
 
 func TestBuildOptions(t *testing.T) {
 	opts := buildOptions(&ServerConfig{})
 
 	_ = opts
 }
+
+func TestPrivateKeyLoading(t *testing.T) {
+	file, err := ioutil.TempFile(testutil.TempDir(), "key")
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer os.Remove(file.Name())
+	key, _, err := crypto.GenerateKeyPair(crypto.RSA, 2048)
+	if err != nil {
+		log.WithError(err).Fatal("Could not generate key")
```

Comment:
Instead of using the log library to fatally crash the test, we should use `t.Fatal` https://golang.org/pkg/testing/#T.Fatal

The logrus output might be swallowed or mangled where the go fatal output is expected and written to the test log.